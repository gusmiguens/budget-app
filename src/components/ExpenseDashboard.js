const React = require('react')
const { ExpenseList } = require('./ExpenseList')
const { ExpenseListFilters } = require('./ExpenseListFilters')

const ExpenseDashboard = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

module.exports = ExpenseDashboard