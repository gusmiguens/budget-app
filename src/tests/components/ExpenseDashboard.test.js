import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashboard from '../../components/ExpenseDashboard'

describe('<ExpenseDashboard />', () => {
    it('Should render ExpenseDashboard correctly', () => {
        const wrapper = shallow(<ExpenseDashboard />)
        expect(wrapper).toMatchSnapshot()
    })

})