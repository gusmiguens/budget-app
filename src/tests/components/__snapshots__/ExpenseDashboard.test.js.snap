// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseDashboard /> Should render ExpenseDashboard correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
